components/side-bar/component-tree/component-status-resolver/component-status-resolver.tsx

```diff
@@ -1,28 +1,36 @@
 import { ComponentID } from '@teambit/component';
 import { ComponentStatus as StatusProps } from '@teambit/workspace';
 import React from 'react';
-
+import classNames from 'classnames';
 import { ComponentStatus } from '../component-status/component-status';
+import { getOverrideColor } from './color-override';
 import { StatusTooltip } from '../component-tooltip';
 import styles from './component-status-resolver.module.scss';
 
 export type ComponentStatusResolverProps = {
   status?: StatusProps;
+  issuesCount?: number;
   id: ComponentID;
 };
 
-export function ComponentStatusResolver({ status, id }: ComponentStatusResolverProps) {
+export function ComponentStatusResolver({ status, id, issuesCount = 0 }: ComponentStatusResolverProps) {
   const isModified = status && (status.modifyInfo.hasModifiedDependencies || status.modifyInfo.hasModifiedFiles);
   if (!status) return null;
-  const colorOverride = isModified ? 'modified' : '';
+  const colorOverride = getOverrideColor(issuesCount, isModified);
   return (
-    <div className={styles.statusLine}>
+    <div className={styles.statusLine} data-tip="" data-for={id?.name}>
+      {issuesCount > 0 && (
+        <div className={classNames(styles.errorBlock, styles.error)}>
+          <span>{issuesCount}</span>,
+        </div>
+      )}
+      //TODO: @odedre need to change Component Status to get status object and then render the ComponentStatus to avoid
```

Comment:
@odedre need to change Component Status to get status object and then render the ComponentStatus to avoid